app/models/adopter_application.rb

```diff
@@ -58,4 +58,8 @@ def self.set_status_to_withdrawn(adopter_application)
     adopter_application.status = :withdrawn
     adopter_application.save
   end
+
+  def applicant_name
```

Comment:
Related to my comment below in `app/models/pet.rb` this method was giving me an error when trying to load applications because the user was retruning `nil` and therefore getting undefined method for nil class. If I added the safe navigator `&` I see that Brandon and Kamala load fine, but any applications from Bad were the issue, as the user was `nil`. I wonder if you have the same issue?